src/actions/guides/database/intro_to_avram_and_orms.cr

```diff
@@ -9,91 +9,124 @@ class Guides::Database::IntroToAvramAndORMs < GuideAction
     <<-MD
     ## Avram
 
-    [Avram](https://github.com/luckyframework/avram/) is the ORM Lucky uses for handling connections to your database, and wrapping the SQL mapping in a friendly Crystal class.
-    An [Object Relational Mapping](https://en.wikipedia.org/wiki/Object-relational_mapping) or "ORM", is a simple way to translate SQL queries, and table schema in to an easy to use object.
+    [Avram](https://github.com/luckyframework/avram/) is the ORM Lucky uses for handling connections
+    to your database, and wrapping the SQL mapping in a friendly Crystal class.
+    An [Object Relational Mapping](https://en.wikipedia.org/wiki/Object-relational_mapping) or "ORM",
+    is a simple way to translate SQL queries, and table schema in to an easy to use object.
 
-    > Avram currently only supports PostgreSQL with no current plans to support any other RDBMS. However, Lucky does not require you to use Avram if your requirements are a bit different.
+    > Avram currently only supports PostgreSQL with no current plans to support any other RDBMS.
+    > However, Lucky does not require you to use Avram if your requirements are a bit different.
```

Comment:
Would we want to link to other ORMs here?

Reply:
There's a lot of them, and some are dying out. Like Crecto was great, but it's no longer maintained. I wouldn't mind showing Granite for people that may need MySQL support, but that can probably come in a different PR.

Reply:
I think we shouldn't  recommend other ORMs  unless one of us actively uses it. I'd worry  that it would be seen as an endorsement. Then the library may not work well with Lucky, may not be maintained, etc. and we'd have a guide pointing to it. So I'm thinking we could leave it as is